app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -67,7 +67,7 @@ public class MainActivity extends CatimaAppCompatActivity implements LoyaltyCard
     private int mLoyaltyCardCount = 0;
     protected String mFilter = "";
     protected Object mGroup = null;
-    protected DBHelper.LoyaltyCardOrder mOrder = DBHelper.LoyaltyCardOrder.Alpha;
+    protected DBHelper.LoyaltyCardOrder mOrder = DBHelper.LoyaltyCardOrder.UsageNumber;
```

Comment:
Please don't change the default sorting style.

Reply:
Done